Sources/Testing/Events/Recorder/Event.Symbol.swift

```diff
@@ -56,27 +56,27 @@ extension Event.Symbol {
   private var _sfSymbolInfo: (privateUseAreaCharacter: Character, name: String) {
     switch self {
     case .default:
-      ("\u{1007C8}", "diamond")
+      ("􀊕", "play.circle")  // running: play.circle (teal)
     case .skip:
-      ("\u{10065F}", "arrow.triangle.turn.up.right.diamond.fill")
+      ("􀺅", "forward.circle")  // skip: forward.circle (purple)
```

Comment:
This property `_sfSymbolInfo` is shared by the current output recorder, right? Since it's the same set of symbols, changing these from (for example) `diamond` to `play.circle` will affect that recorder right away, even though your new `AdvancedConsoleOutputRecorder` is not yet enabled by default. So I think if you want to modify these, you'll need to duplicate this property and use your new symbols in that copy.